pymc3/step_methods/hmc/quadpotential.py

```diff
@@ -472,6 +480,7 @@ def __init__(
         initial_cov=None,
         initial_weight=0,
         adaptation_window=101,
+        adaptation_window_multiplier=1,
```

Comment:
this is a bit annoying, but i'd get rid of the `doubling` parameter, update the docstring above, and _probably_ keep this defaulted to 2?

Reply:
All done!